guidance/models/_openai.py

```diff
@@ -1,26 +1,19 @@
-from typing import TYPE_CHECKING, Callable, Iterator, Optional, Union
+from typing import Optional
 
-from pydantic import TypeAdapter
 
-
-from .._ast import (
-    JsonNode,
-    RuleNode,
-)
-from ..trace import OutputAttr
 from ._base import Model
-
 from ._openai_base import (
     BaseOpenAIInterpreter,
-    AudioContent,
-    OpenAIState,
     Message,
-    OpenAIImageMixin,
     OpenAIAudioMixin,
+    OpenAIImageMixin,
+    OpenAIJSONMixin,
+    OpenAIRegexMixin,
+    OpenAIRuleMixin,
 )
 
 
-class OpenAIInterpreter(BaseOpenAIInterpreter):
+class OpenAIInterpreter(OpenAIRuleMixin, OpenAIJSONMixin, OpenAIRegexMixin, BaseOpenAIInterpreter):
```

Comment:
This feels slightly ridiculous, but I don't think that the code in these Mixins should be in the vLLM interpreter